src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html

```diff
@@ -127,7 +127,7 @@ <h3 class="document-type">
 </div>
 <div class="row py-16" id="disclaimer">
     <p class="caption my-8" data-hide-on-it="true">L’attestation indique la date à laquelle le destinataire (ou son délégué) a eu accès pour la première fois, via la plateforme, à l’acte à notifier. Cette date ne correspond pas nécessairement à la date de finalisation de la notification, qui peut également être antérieure en raison de l'expiration du délai. Afin d’identifier la date correcte de réception de la notification, le destinataire (ou son délégué) est invité à vérifier <strong><a class="faq-link-perfezionamento" href="https://notifichedigitali.pagopa.it/fr/faq/#perfezionamento-come-posso-sapere">dans cette FAQ</a></strong> comment doit être calculée la date de réception.</p>
-    <p class="caption my-8">L’attestazione riporta la data in cui il destinatario (o il suo delegato) per la prima volta ha avuto accesso tramite la piattaforma al documento oggetto di notificazione. Tale data non necessariamente corrisponde alla data di perfezionamento della notifica, che può anche essere antecedente per decorrenza termini. Ai fini di individuare la giusta data di perfezionamento della notifica, si invita il destinatario (o il suo delegato) a verificare <strong><a class="faq-link-perfezionamento" href="https://notifichedigitali.pagopa.it/faq#perfezionamento-come-posso-sapere">in questa FAQ</a></strong> come deve essere calcolata la data di perfezionamento.</p>
+    <p class="caption my-0">L’attestazione riporta la data in cui il destinatario (o il suo delegato) per la prima volta ha avuto accesso tramite la piattaforma al documento oggetto di notificazione. Tale data non necessariamente corrisponde alla data di perfezionamento della notifica, che può anche essere antecedente per decorrenza termini. Ai fini di individuare la giusta data di perfezionamento della notifica, si invita il destinatario (o il suo delegato) a verificare <strong><a class="faq-link-perfezionamento" href="https://notifichedigitali.pagopa.it/faq#perfezionamento-come-posso-sapere">in questa FAQ</a></strong> come deve essere calcolata la data di perfezionamento.</p>
```

Comment:
it's better to remove only margin bottom and leave mt-8

Reply:
done

Reply:
use `mb-0` because `my-0` doesn't exist in css